src/main/kotlin/org/rust/ide/annotator/RsSyntaxErrorsAnnotator.kt

```diff
@@ -43,6 +45,7 @@ class RsSyntaxErrorsAnnotator : AnnotatorBase() {
             is RsValueParameter -> checkValueParameter(holder, element)
             is RsTypeParameterList -> checkTypeParameterList(holder, element)
             is RsTypeArgumentList -> checkTypeArgumentList(holder, element)
+            is RsConstant -> checkConstantType(holder, element)
```

Comment:
It seems this branch is unreachable since `RsConstant` is `RsItemElement`

Reply:
Indeed, I forgot to remove it. Thank you :)